public/src/css/no-more-tables.css

```diff
@@ -0,0 +1,49 @@
+@media only screen and (max-width: 800px) {
+    
+    /* Force table to not be like tables anymore */
+	#no-more-tables table, 
```

Comment:
No need separate css file for that.
`no-more-tables` is bad name, maybe `table-fluid` or `table-mobile`? Can't we use some Bootstrap native features for that?